pkg/lib/global/global.go

```diff
@@ -0,0 +1,4 @@
+package global
```

Comment:
should we choose a different name? common? shared?
root/<gloval or shared or common>